Cold-Gas-Thruster/CGT-PDD.bib

```diff
@@ -0,0 +1,11 @@
+@article{nozzles,
```

Comment:
I remember putting several citation needed comments in my last review but where are the citations?

I'm surprised that you only have one document to reference.